Marlin/src/module/motion.h

```diff
@@ -320,8 +320,8 @@ void homeaxis(const AxisEnum axis);
               mirrored_duplication_mode;          // Used in Dual X mode 3
 #endif
 
-#if ENABLED(MULTI_NOZZLE_DUPLICATION) && HOTENDS > 2
-  uint8_t duplication_e_mask;
+#if EITHER(DUAL_X_CARRIAGE, MULTI_NOZZLE_DUPLICATION)
+  extern uint8_t duplication_e_mask;
```

Comment:
Well, this is clearly not correct. I'm following your guidance as the seed for my changes here. `duplication_e_mask` is not needed in `DUAL_X_CARRIAGE` mode. It is only needed for `MULTI_NOZZLE_DUPLICATION` (where there are more than two extruders). More than two, because you are either using Tool 0, Tool 1, or Both, so no fancy bitmask is required.